hyperkube/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#
+# Creates hyperkube with CNI binaries and flannel configuration
+#
+
+FROM gcr.io/google_containers/hyperkube-amd64:v1.3.0-beta.1
+RUN apt-get update && apt-get upgrade
+RUN apt-get install xz-utils
+
+RUN mkdir -p /opt/cni/bin
+RUN mkdir -p /etc/cni/net.d
+RUN mkdir -p /var/run/flannel
+
+WORKDIR /tmp
+
+RUN curl -L https://github.com/containernetworking/cni/releases/download/v0.3.0/cni-v0.3.0.txz -O
+RUN tar xvf cni-v0.3.0.txz -C /opt/cni/bin
+RUN rm cni-v0.3.0.txz
+
+COPY 10-containernet.conf /etc/cni/net.d
```

Comment:
You need a second conf file that contains the loopback configuration - like in the samples. It might work without conf file, because when creating network namespace manually it is created with loopback device. But I have not tried it and I am not sure if there is any difference.